ros/api/v0/hosts.py

```diff
@@ -129,7 +130,9 @@ def get(self):
                 system_dict = row.System.__dict__
                 host = {skey: system_dict[skey] for skey in SYSTEM_COLUMNS}
                 host['account'] = row.RhAccount.account
-                host['performance_utilization'] = row.PerformanceProfile.performance_utilization
+                host['performance_utilization'] = sort_io_dict(
+                    row.PerformanceProfile.performance_utilization
+                )
```

Comment:
I would prefer to keep method inside model `PerformanceProfile` than in utils which returns `performance_utilization` value with sorted `io_all`.

Reply:
@kgaikwad I think I would like to address it on a different PR